### ISSUE TITLE
Setup Failure

### ISSUE THREAD

**Issue by ryantstone** [Blaked84/heroku-alfred-workflow#1]
When trying to update the app I'm getting a failure that it's unable to find Alfred workflow gem. Haven't gotten a chance to look at it further but I'll work on it this weekend and open a PR.

From Alfred Debug:
```
Code 1: /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- alfred-3_workflow (LoadError)
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /Users/sharkmaul/Dropbox (Personal)/Documents/Scripts/Alfred.alfredpreferences/workflows/user.workflow.F67B7EFB-E342-4E34-BDD1-16361983EEFC/heroku.rb:2:in `<main>'
```

**Comment by Blaked84** [comment]
The Alfred workflow gem is installed with `bundle install --standalone` but it's not the best solution as it's installed for only one ruby version, here 2.4.

I'll take a look too,thank you for your feedback!

**Comment by kehphin** [comment]
I'm also getting this error - tried manually putting the gem in `/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/gems/2.3.0/gems` but the folder the `/System` folder is read-only :/

**Comment by Blaked84** [comment]
Could you try this version? https://github.com/Blaked84/heroku-alfred-workflow/blob/remove-gems/Heroku.alfredworkflow

The issue should be solved, I moved gems files in the project. I'm not satisfied with this workaround but it avoids to install gems system wide if you don't use rbenv or rvm.

**Comment by Blaked84** [comment]
I found a related issue with ruby 2.3.0, it is fixed (057385d4d3553a9a3eb4bfe59910e2d175f23fe9) now in https://github.com/Blaked84/heroku-alfred-workflow/releases/download/untagged-45ad5b6af89753337735/Heroku.alfredworkflow
